src/utils/__tests__/paypal.test.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable functional/immutable-data */
+// eslint-disable-next-line no-underscore-dangle
+(window as any)._env_ = {
+  WALLET_CONFIG_API_BASEPATH: "/webview-payment-wallet/v1",
+  WALLET_CONFIG_API_PM_BASEPATH: "/pp-restapi-CD",
+  WALLET_OUTCOME_API_BASEPATH: "/payment-wallet",
+  WALLET_CONFIG_API_TIMEOUT: "10000",
+  WALLET_CONFIG_API_ENV: "DEV",
+  WALLET_CONFIG_API_HOST: "https://api.dev.platform.pagopa.it",
+  WALLET_CONFIG_WEBVIEW_PM_HOST: "https://api.dev.platform.pagopa.it",
+  WALLET_NPG_SDK_URL:
+    "https://stg-ta.nexigroup.com/monetaweb/resources/hfsdk.js",
+  WALLET_PAGOPA_LOGOS_CDN: "https://assets.cdn.io.italia.it/logos/abi"
+};
```

Comment:
These variables can be removed because now they are globally available:
https://github.com/pagopa/pagopa-wallet-fe/blob/main/jest.config.js#L16